download-data.ts

```diff
@@ -0,0 +1,44 @@
+import { $ } from "zx";
```

Comment:
Move this file to a new top-level directory called `scripts`

Reply:
done